system/ee/ExpressionEngine/Controller/Publish/AbstractPublish.php

```diff
@@ -372,7 +372,17 @@ protected function validateEntry(ChannelEntry $entry, $layout)
             }
         }
 
-        $entry->set($_POST);
+        if (defined('CLONING_MODE') && CLONING_MODE === true && $this->entryCloningEnabled($entry)) {
+            $entry->setId(null);
+            while (true !== $entry->validateUniqueUrlTitle('url_title', $_POST['url_title'], ['channel_id'], null)) {
+                $_POST['url_title'] .= '_1';
```

Comment:
I think we need to check to confirm this is actually unique... Thinking people will likely use the same entry as a base multiple times.. tagging @intoeetive

Reply:
Notice we have this in `while` loop. It is assuring the url_title to be unique